lib/browser/message-port-main.ts

```diff
@@ -0,0 +1,25 @@
+import { EventEmitter } from 'events'
+
+export class MessagePortMain extends EventEmitter {
+  _internalPort: any
```

Comment:
Is it worth making this a v8 hidden or otherwise hiding this from end users, or is it ok that this is theoretically usable